### PR TITLE
[lab-Machine learning] Rui Almeida

### PR DESCRIPTION
**Pull request by RuiAlmeida97**

### CODE REVIEW

**Comment by ta-data-lis** [comment]
Lab Checked ✅
**Challenge5** *Now that we have split the data to predictor and response variables and imported the train_test_split() function, split X and y into X_train, X_test, y_train, and y_test. 80% of the data should be in the training set and 20% in the test set. train_test_split()*
Don't forget to specify your test_size. If you only want 20% for htest then:
```
X_train, X_test, y_train, y_test = train_test_split(X,y,random_state=0,test_size=0.20)
```
Nice work! Keep it up 👍
